apps/web/src/styles/file-browser.css

```
@@ -519,4 +524,132 @@
+/* Item checkbox */
+.file-list-item-checkbox {
+  display: none;
+  cursor: pointer;
+  font-family: var(--font-family-mono);
+  font-size: var(--font-size-sm);
+  color: var(--color-text-secondary);
+  margin-right: 4px;
+  flex-shrink: 0;
+}
+
+.file-list-item-checkbox--visible {
```

Stylelint (17.2.0)

[error] Expected class selector ".file-list-item-checkbox--visible" to be kebab-case (selector-class-pattern)

(selector-class-pattern)